src/Spryker/Client/RabbitMq/Model/Connection/ConnectionConfigFilter/ConnectionConfigFilter.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Spryker\Client\RabbitMq\Model\Connection\ConnectionConfigFilter;
+
+use Generated\Shared\Transfer\QueueConnectionTransfer;
+use Spryker\Client\RabbitMq\Dependency\Client\RabbitMqToStoreClientInterface;
+use Spryker\Client\RabbitMq\Model\Connection\ConnectionFactoryInterface;
+
+class ConnectionConfigFilter implements ConnectionConfigFilterInterface
+{
+    /**
+     * @var \Spryker\Client\RabbitMq\Dependency\Client\RabbitMqToStoreClientInterface
+     */
+    protected $storeClient;
+
+    /**
+     * @var \Spryker\Client\RabbitMq\Model\Connection\ConnectionFactoryInterface
+     */
+    protected $connectionFactory;
+
+    /**
+     * @param \Spryker\Client\RabbitMq\Dependency\Client\RabbitMqToStoreClientInterface $storeClient
+     * @param \Spryker\Client\RabbitMq\Model\Connection\ConnectionFactoryInterface $connectionFactory
+     */
+    public function __construct(RabbitMqToStoreClientInterface $storeClient, ConnectionFactoryInterface $connectionFactory)
+    {
+        $this->storeClient = $storeClient;
+        $this->connectionFactory = $connectionFactory;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\QueueConnectionTransfer[] $connectionsConfig
+     * @param string|null $locale
+     *
+     * @return \Generated\Shared\Transfer\QueueConnectionTransfer[]
+     */
+    public function filterByLocale(array $connectionsConfig, ?string $locale): array
```

Comment:
$locale - Is it localeCode/localeName? It seems to me it would be clearer to have such name (everywhere)

Reply:
I guess it is a locale code. Could you give an example what is locale code and what is locale name in your opinion?

Reply:
Done